app/presenters/topics/show_presenter.rb

```diff
@@ -57,7 +57,7 @@ def replies
     end
 
     def details_of_replies
-      replies.map { |reply| reply.as_json(POST_FIELDS).merge(like_data(reply).as_json) }
+      replies.map { |reply| reply.as_json(POST_FIELDS).merge(like_data(reply).as_json).merge({ edited_at: reply.text_versions.last&.updated_at }.as_json) }
```

Comment:
Shouldn't the `replies` method be updated to preload `text_version` to avoid N+1? 🤔 

```ruby
def replies
  ActiveRecord::Precounter.new(
    @topic.replies.live.includes(replies: :text_versions)
  ).precount(:post_likes)
end
```

☝️  FYI, I haven't tested this.